config/jobs/kubernetes/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -723,3 +723,80 @@ periodics:
     testgrid-tab-name: ci-cloud-provider-azure-multiple-zones
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: "Runs Kubernetes multiple availability zones tests with cloud-provider-azure (https://github.com/kubernetes/cloud-provider-azure) on VMSS."
+
+# scalability tests
+- interval: 3h
+  # ci-kubernetes-kubemark-100-azure runs kubemark tests with cloud-provider-azure of 100 nodes.
+  name: ci-kubernetes-kubemark-100-azure
+  tags:
+  - "perfDashPrefix: kubemark-100Nodes"
+  - "perfDashJobType: performance"
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+    preset-e2e-kubemark-common: "true"
+  annotations:
+    testgrid-dashboards: provider-azure-scalability
+    testgrid-tab-name: ci-kubernetes-kubemark-100-azure
+    testgrid-num-failures-to-alert: '1'
+    testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
+    description: "Runs kubemark tests (100 nodes) with the Azure cloud provider enabled."
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20191031-df9cbb4-master
+      args:
+      - --job=$(JOB_NAME)
+      - --repo=k8s.io/kubernetes=release-1.16
+      - --repo=k8s.io/$(REPO_NAME)=master
+      - --root=/go/src
+      - --service-account=/etc/service-account/service-account.json
+      - --timeout=260
+      - --upload=gs://kubernetes-jenkins/logs/
+      - --scenario=kubernetes_e2e
+      - --
+      - --cluster=kubemark-100
+      - --extract=ci/latest
+      - --deployment=acsengine
+      - --build=bazel
+      - --provider=skeleton
+      - --ginkgo-parallel=30
+      - --acsengine-agentpoolcount=2
+      - --acsengine-admin-username=azureuser
+      - --acsengine-creds=$AZURE_CREDENTIALS
+      - --acsengine-orchestratorRelease=1.16
+      - --acsengine-mastervmsize=Standard_DS2_v2
+      - --acsengine-agentvmsize=Standard_D4s_v3
```

Comment:
we may need bigger size for this test, but let's run some jobs before determining the final vm size

Reply:
OK. Let's wait and see.